<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>439</width>
         <height>491</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSchedule" name="edt" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_dition">
    <property name="title">
     <string>Ã‰dition</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu_dition"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>274</width>
     <height>511</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>75</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Classroom</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
<<<<<<< HEAD
          <widget class="QComboBox" name="comboBoxClassroom">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>30</height>
=======
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
>>>>>>> 147c2b0285400e32547caa52d601089228d27cb9
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Module</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
<<<<<<< HEAD
          <widget class="QComboBox" name="comboBoxModule">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>30</height>
=======
          <widget class="QComboBox" name="comboBox_2">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
>>>>>>> 147c2b0285400e32547caa52d601089228d27cb9
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Group</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
<<<<<<< HEAD
          <widget class="QComboBox" name="comboBoxGroup">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>30</height>
=======
          <widget class="QComboBox" name="comboBox_3">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
>>>>>>> 147c2b0285400e32547caa52d601089228d27cb9
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>200</height>
        </size>
       </property>
       <property name="firstDayOfWeek">
        <enum>Qt::Monday</enum>
       </property>
       <property name="horizontalHeaderFormat">
        <enum>QCalendarWidget::NoHorizontalHeader</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="addTimeSlotButton">
       <property name="text">
        <string>Add Time Slot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSchedule</class>
   <extends>QWidget</extends>
   <header>QSchedule.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
